mlocal/frags/Makefile.stub

```diff
@@ -77,6 +81,18 @@ $(go_OBJ): $(BUILDDIR)/vendors-done $(libruntime) $(BUILDDIR)/config.h
 	@rm -f $(go_OBJ)
 	$(V)export BUILDDIR=`readlink -f $(BUILDDIR)` && cd $(SOURCEDIR)/src/pkg/buildcfg && go generate
 
+# bash_completion file
+$(bash_completion): $(go_OBJ)
+	@rm -f $(bash_completion)
+	@mkdir -p $(@D)
+	$(V) go run  $(SOURCEDIR)/etc/bash_completion.d/bash_completion.go $(bash_completion)
+
+# install singularity CLI bash_completion file
+$(bash_completion_INSTALL): $(bash_completion)
+	@echo " INSTALL" $@
+	$(V)install -d $(@D)
+	$(V)install -m 0644 $(bash_completion) $(bash_completion_INSTALL)
```

Comment:
Here change $(V)install -m 0644 $(bash_completion) $(bash_completion_INSTALL) for : 
$(V)install -m 0644 $< $@